tensorrt_llm/_torch/auto_deploy/transform/library/mxfp4_moe.py

```
@@ -151,38 +174,260 @@ def _get_topk_from_router(node: Node) -> int:
+    if not isinstance(tensor, torch.Tensor):
+        raise TypeError(f"Expected get_attr target {target} to be a tensor, got {type(tensor)}.")
+    return tensor
+
+
+def _mxfp4_target_names_from_node(node: Node) -> dict[str, str] | None:
+    if not is_op(node, _MXFP4_RUNTIME_OPS):
+        return None
+
+    expert_args = extract_op_args(node, *_MXFP4_LAYOUT_ARG_NAMES)
+    target_names: dict[str, str] = {}
+    for name, arg in zip(_MXFP4_LAYOUT_ARG_NAMES, expert_args):
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)